src/components/displaybook.js

```diff
@@ -1,13 +1,79 @@
-/* eslint-disable react/prop-types */
-export default function Book({ book }) {
+import React, { useEffect } from 'react';
+import '../styles/bookdisplay.css';
+import { useSelector, useDispatch } from 'react-redux';
+import { getBookItems, removeBook } from '../redux/books/booksSlice';
+
+const Book = () => {
+  const dispatch = useDispatch();
+  const { books, isLoading, error } = useSelector((state) => state.books);
+
+  useEffect(() => {
+    dispatch(getBookItems());
+  }, [dispatch]);
+
+  const renderBook = (itemId, book) => {
+    const { category, title, author } = book;
+    return (
+      <div key={itemId} className="containerRender" style={{ display: 'flex' }}>
```

Comment:
:red_circle: I've reviewed your design and I noticed a few inconsistencies with the wireframe.Kindly check the table below :arrow_down: 

|Your Design|[Zeplin Design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378)|
|:----:|:----:|
|![fix](https://github.com/ethred/book-store-React/assets/26736582/32c115c9-17cf-4b38-8bdb-47fb1d4a4e5e)|![good](https://github.com/ethred/book-store-React/assets/26736582/1f59fa67-3341-406e-a858-46ebb60d2744)|
|![no-progress](https://github.com/ethred/book-store-React/assets/26736582/093afe18-a9f0-4963-821d-4224d339b7d6)|![progrss](https://github.com/ethred/book-store-React/assets/26736582/c0c94d43-f197-4bf0-8d44-f775cde594ee)|
|![urdesign](https://github.com/ethred/book-store-React/assets/26736582/f67faeca-3afe-4156-bd8d-2674ba710caf)|![zilphin](https://github.com/ethred/book-store-React/assets/26736582/39147db5-cc7c-4ae7-a7c6-057601c720f1)|